blog/2025-2-15-pixi-v8-launches.md

```diff
@@ -0,0 +1,228 @@
+- **Boosting Performance:** PixiJS is already renowned for its speed. With v8, we've unlocked even greater performance, making it faster across the board compared to v7.
+
+We're incredibly proud of PixiJS v8 and eager to share the improvements and new features with you. While there are some breaking API changes, we've provided a migration guide and ensured compatibility with v7 wherever possible. Get ready to experience the next level of 2D rendering with PixiJS v8!
+
+---
+
+## 🔗 Quick links
+- The new Docs for v8 can be found here [LINK TO DOCS]
+- [Migration](https://pixijs.com/guides/migrations/v8)
+- [Examples](https://pixijs.com/examples)
+- [Open Games](https://github.com/pixijs/open-games)
+
+---
+
+## 🎁 Whats New?
+
+So many new things - more than we can easily cover in one post! Here's some head lines, but check out the links above for a more in depth details on the changes!
+#### 📦 New Package Structure
+
+No more "lerna." PixiJS is now just one package with one import root: `import {stuff} from ‘pixi.js’`. This change means we now have much better tree shaking during app compilation, reducing bundle size if not imported.
+
+Old:
+
+```ts
+import { Sprite } from "@pixi/sprite";
+import { Graphic } from "@pixi/graphics";
+```
+
+New:
+
+```ts
+import { Sprite, Graphic } from "pixi.js";
+```
+
+### ✨ We *promise* the Renderer will work
+
+When initializing a renderer, this process is now asynchronous. This serves two purposes: firstly, identifying and loading the necessary renderer code to minimize what is loaded for your users. We only load the one backend that your user is using. There's no point in loading all the WebGL stuff if they are using WebGPU!
+
+Secondly, the initialization of WebGPU itself is an asynchronous process, so we need to have a promise in there somewhere :)
+
+```ts
+import { Application, autoDetectRenderer } from "pixi.js";
+
+const app = new Application();
+
+(async () => {
+  await app.init({
+    // application options
+  });
+
+  // or
+  const renderer = await autoDetectRenderer({}); // WebGL or WebGPU
+
+  // do pixi things
+})();
+```
+
+#### 🖥️ WebGPU Renderer
+
+![PixiJS + webGPU = love](image-1.png)
+
+The obvious one! We have a WebGPU backend to render things. It's worth noting that WebGPU is not inherently faster in all use cases than WebGL (nor is it slower). This is because we are generally more CPU-bound in PixiJS than GPU-bound. However, if you find that your scene has lots of batch breaks (e.g., filters, masks, blend modes), you may find WebGPU will perform faster - especially from the GPU side. WebGPU is still new, and I predict it will get faster with time, just as WebGL did! It's a great starting point!
+
+#### 📈 New Performance Bar
```

Comment:
In journalism this is called "burying the lede". I think You should lead with WebGPU and performance. It's the most impressive and important part of this story. More so than the package structure or Promise-ifying Application/autoDetectRenderer above.

Reply:
good point! will shift them up!